lib/formatter/csv/annotation_for_csv.rb

```diff
@@ -6,34 +6,87 @@ class AnnotationForCsv
       def initialize(classification, annotation, cache)
         @classification = classification
         @annotation = annotation.dup.with_indifferent_access
+        @current = @annotation.dup
         @cache = cache
       end
 
       def to_h
-        annotation.merge! "task_label" => task_label
+        send(task['type'])
```

Comment:
I added a whitelist in the form of a private method with an array of valid task types to check against. Also, an error hash to jsonify and include in the csv if the task is invalid. I think that should cover us. If you'd rather go back to case/when, that's cool. This also seemed to help get some of the failing Formatter::Csv::Classification specs passing, woop.